code/modules/projectiles/boxes_magazines/external/pistol.dm

```diff
@@ -83,3 +83,16 @@
 	caliber = ".50"
 	max_ammo = 7
 	multiple_sprites = AMMO_BOX_PER_BULLET
+
+/obj/item/ammo_box/magazine/m9mm_mkshft
+	name = "makeshift pistol magazine (9mm)"
+	icon_state = "9x19mks-6"
+	desc= "A makeshift pistol mag, made out of spare parts. Doesn't look too reliable."
+	ammo_type = /obj/item/ammo_casing/c9mm
+	caliber = "9mm"
+	max_ammo = 6
+	start_empty = TRUE
+
+/obj/item/ammo_box/magazine/m9mm_mkshft/update_icon()
+	..()
+	icon_state = "9x19mks-[ammo_count() ? "6" : "0"]"
```

Comment:
Newline or travis will spill milk on your shoes.

Reply:
Yeah fixed that just now